aioxmpp/presence/service.py

```diff
@@ -438,3 +591,66 @@ def resend_presence(self):
 
         if self.client.established:
             return self.client.enqueue(self.make_stanza())
+
+    def subscribe_peer_directed(self,
+                                peer: aioxmpp.JID,
+                                muted: bool = False):
+        """
```

Comment:
I believe this should take an initial presence as optional argument (so that you can already send a modified directed presence with the request).

Reply:
That’s what ``muted=True`` is for.

Reply:
My argument was less about the capability, but more about convenience of use, and conciseness of the code.

Reply:
Right, but then I’d prefer to be able to set a ``presence_filter`` via the interface.